scripts/combine/setupCombine.py

```diff
@@ -638,21 +652,32 @@ def assertSample(name, startsWith=["W", "Z"], excludeMatch=[]):
 
     # Experimental range
     #widthVars = (42, ['widthW2p043GeV', 'widthW2p127GeV']) if wmass else (2.3, ['widthZ2p4929GeV', 'widthZ2p4975GeV'])
-    # Variation from EW fit (mostly driven by alphas unc.)
-    widthVars = (0.6, ['widthW2p09053GeV', 'widthW2p09173GeV']) if wmass else (0.8, ['widthZ2p49333GeV', 'widthZ2p49493GeV'])
-    cardTool.addSystematic(f"widthWeight{label}",
-                            rename=f"Width{label}{str(widthVars[0]).replace('.','p')}MeV",
-                            processes=["signal_samples_inctau"],
-                            action=lambda h: h[{"width" : widthVars[1]}],
-                            group=f"width{label}",
+    # Variation from EW fit (mostly driven by alphas unc.)    
+    cardTool.addSystematic(f"widthWeightZ",
+                            rename=f"WidthZ0p8MeV",
+                            processes= ['single_v_nonsig_samples'] if wmass else signal_samples_forMass,
+                            action=lambda h: h[{"width" : ['widthZ2p49333GeV', 'widthZ2p49493GeV']}],
+                            group=f"widthZ",
```

Comment:
Since it is a negligible effect, can we put this in the same group as the Z mass shift ? Also renaming that group from "massShift" to something more specific, like "ZmassShape" or "ZmassAndWidth"